allennlp/models/bidirectional_lm.py

```diff
@@ -262,7 +262,8 @@ def forward(self,  # type: ignore
             return_dict = {
                     'loss': average_loss * scale_factor,
```

Comment:
One outstanding issue that I couldn't think of a good way to handle is adjusting for the `scale_factor`. In Calypso the adjustment is made in `train.py`, but that only works because that code knows it's dealing with a specific model. Two questions:

1. What is the purpose behind `scale_factor`? Can/should we drop it?
2. If not, can you think of clean way to make it work with `"batch_weight"`?

Reply:
Thanks, dropped!